examples/oft-aptos/hardhat.config.ts

```diff
@@ -0,0 +1,93 @@
+// Get the environment configuration from .env file
+//
+// To make use of automatic environment setup:
+// - Duplicate .env.example file and name it .env
+// - Fill in the environment variables
+import 'dotenv/config'
+
+import 'hardhat-deploy'
+import 'hardhat-contract-sizer'
+import '@nomiclabs/hardhat-ethers'
+import '@layerzerolabs/toolbox-hardhat'
+import { HardhatUserConfig, HttpNetworkAccountsUserConfig } from 'hardhat/types'
+
+import { EndpointId } from '@layerzerolabs/lz-definitions-v3'
+
+// Set your preferred authentication method
+//
+// If you prefer using a mnemonic, set a MNEMONIC environment variable
+// to a valid mnemonic
+const MNEMONIC = process.env.MNEMONIC
+
+// If you prefer to be authenticated using a private key, set a PRIVATE_KEY environment variable
+const PRIVATE_KEY = process.env.PRIVATE_KEY
+
+const accounts: HttpNetworkAccountsUserConfig | undefined = MNEMONIC
+    ? { mnemonic: MNEMONIC }
+    : PRIVATE_KEY
+      ? [PRIVATE_KEY]
+      : undefined
+
+if (accounts == null) {
+    console.warn(
+        'Could not find MNEMONIC or PRIVATE_KEY environment variables. It will not be possible to execute transactions in your example.'
+    )
+}
+
+const config: HardhatUserConfig = {
+    paths: {
+        cache: 'cache/hardhat',
+    },
+    solidity: {
+        compilers: [
+            {
+                version: '0.8.22',
+                settings: {
+                    optimizer: {
+                        enabled: true,
+                        runs: 200,
+                    },
+                },
+            },
+        ],
+    },
+    networks: {
+        'eth-testnet': {
+            eid: EndpointId.ETHEREUM_V2_TESTNET,
+            url: process.env.RPC_URL_ETH_TESTNET || 'http://127.0.0.1:8501',
+            accounts,
+        },
+        'bsc-testnet': {
+            eid: EndpointId.BSC_V2_TESTNET,
+            url: process.env.RPC_URL_BSC_TESTNET || 'http://127.0.0.1:8502',
+            accounts,
+        },
+        'aptos-testnet': {
+            eid: EndpointId.APTOS_V2_TESTNET,
+            url: process.env.RPC_URL_APTOS_TESTNET || 'http://127.0.0.1:8080',
+            accounts,
+        },
+        'sandbox-eth': {
```

Comment:
I think it would be better if we didnt include sandboxes in this example

Reply:
still under dev. using sandboxes for it